src/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { UFO } from './ufo'
+import { ParamsObject } from './parse'
+
+export function withoutTrailingSlash (input: string = ''): string {
+  return input.endsWith('/') ? input.slice(0, -1) : input
```

Comment:
Nitpicking, what about using regexp for the utils? replace(/\/$/, ‘’)

Reply:
I wasn't sure about perf differences but seems better idea with regex

Reply:
If it’s only for start or end character, I think it’s performance should OK

Reply:
Indeed... Still keeping it since it avoid unnecessary regex and memory allocation when string matches (since we also have with* variants)